db/migrate/20230308160615_create_data_resources.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class CreateDataResources < ActiveRecord::Migration[7.0]
+  def change
+    create_table :data_resources, id: :uuid do |t|
+      t.text :identifier
+      t.text :title
+      t.text :description
+      t.text :keywords, array: true, default: []
+      t.text :theme
+      t.text :license
+      t.text :version
+      t.text :contact_name
+      t.text :contact_email
+      t.text :alternative_title
```

Comment:
alternative title should also be an array, and let's make it `alternative_titles`